packages/universal_io/lib/src/constants/operating_system_constants.dart

```diff
@@ -0,0 +1,20 @@
+/// Holds the operating system [String]s.
+class OS {
```

Comment:
```suggestion
class OperatingSystem {
```
Should the file name match the name of this class?